app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<section class="section">
+    <div class="container">
+        <div class="columns is-centered">
+            <div class="column is-half">
+                <div class="field is-grouped is-justify-content-right">
+                    <p class="control">
+                        <%= link_to new_recipe_path do %>
+                            <button 
+                            class="button js-modal-trigger is-save is-outlined is-rounded is-align-content-end" 
+                            data-target="modal-js-example">
+                            New Recipe
+                            </button>
+                        <% end %>  
+                    </p>
+                </div>
+
+                <% @recipes.each do | recipe | %>
+                    <div class="box">
+                        <article class="media">
+                            <div class="media-left">
+                                <figure class="image is-64x64">
+                                    <img src="https://via.placeholder.com/64.png">
+                                </figure>
+                            </div>
+                            <div class="media-content">
+                                <div class="content">
+                                    <strong><%= recipe.name %></strong><br />
+                                    <p><%= recipe.description %></p>
```

Comment:
- [ ] I noticed that you are listing all the recipes in the database, however, the recipes listed here should belong to the logged in user only. Kindly make this minor change 👍

Reply:
Thanks @codepantha for you time, we already implemented this rule. Please give it another look.